### ISSUE TITLE
[TODO] Add a 200-250px right sidebar

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/reactsocialnetwork#82]
**Is your feature request related to a problem? Please describe.**

No. This is not related to a problem. Adding a feature. 

Add a 200-250px right sidebar for future use as nav, alerts, notifications, etc.

Also we should be using more effective semantic HTML tags like main, aside, article, etc. Right now most of the app is a section.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by promise-dash** [comment]
Hey, I would like to solve this issue.
Could you please elaborate a bit on which component you want to add the right sidebar

**Comment by gbowne1** [comment]
My bad @promise-dash 

This is issue is already completed.  We now have a sidebar.  There are items we need to add to it, but, this issue was meant for creating the sidenav itself and thats already completed 

Closing this as completed.
